timely_beliefs/beliefs/utils.py

```diff
@@ -647,6 +647,18 @@ def read_csv(
     return classes.BeliefsDataFrame(df, sensor=sensor)
 
 
+def find_out_extension(path: str):
+    if isinstance(path, str):
+        ext = path.split(".")[-1]
+    elif hasattr(path, "filename"):
+        # For example, supports werkzeug.datastructures.FileStorage objects
+        ext = path.filename.split(".")[-1]
+    else:
+        # Given this function's name, we'll let Pandas attempt to read the path as a CSV file
+        ext = "csv"
+    return ext
```

Comment:
We don't do anything with `ext` but return it, so the function can be a bit more straightforward (return right away).